recipes/newrelic/infrastructure/kubernetes.yml

```diff
@@ -339,6 +339,13 @@ install:
               fi
             fi
 
+            if helm version --short --client | grep 'v2' > /dev/null; then
+              echo "Helm version 2.x detected. Support for Helm v2 will be removed in the near future,"
+              echo "please upgrade at your earliest convenience."
+              echo "Will install Kubernetes Integration v2.8.3, last version compatible with Helm v2."
+              ARGS="${ARGS} --version 3.2.11"
```

Comment:
Please note that the difference between these two versions is intended. Here we're pinning the chart version, which includes the aforementioned nri-kubernetes version:

```shell
helm template newrelic-bundle newrelic/nri-bundle --version 3.2.11 --set global.cluster=x --set global.licenseKey=x | grep 'newrelic/infrastructure-k8s'
          image: "newrelic/infrastructure-k8s:2.8.3"
```